graylog2-web-interface/src/components/content-packs/ContentPackSelection.test.jsx

```diff
@@ -43,25 +44,17 @@ describe('<ContentPackSelection />', () => {
   it('should update the state when filling out the form', () => {
     let called = 0;
     const changeFn = jest.fn((state) => {
-      const contentPack = {
-        name: 'name',
-        summary: 'summary',
-        description: 'descr',
-        vendor: 'vendor',
-        url: 'url',
-      };
       called += 1;
-      if (called === Object.keys(contentPack).length) {
-        expect(state.contentPack).toEqual(contentPack);
+      if (called === 5) {
```

Comment:
What happens when one of the change callbacks does not fire? The following `expect()` calls will never be executed then.

Reply:
So this expect will fail:
`expect(changeFn.mock.calls.length).toBe(5);`

Reply:
Why don't you check the state of `state.contentPack` immediately after that expect? To me it looks like there is an unnecessary level of asynchronicity.